Classes/Task/Task.php

```diff
@@ -0,0 +1,67 @@
+<?php
+require_once 'Maps/TaskData.php';
+
+class Task
+{
+    const STATUS_NEW = 'Новое задание';
+    const STATUS_CANCELED = 'Задание отмененно';
+    const STATUS_IN_WORK = 'В работе';
+    const STATUS_DONE = 'Выполнено';
+    const STATUS_FAILED = 'Провалено';
+
+    public $status;
+    public $authorID;
+    public $description;
+    public $taskPerformerId;
+
+    public function __construct($authorID, $description)
+    {
+        $this->authorID = $authorID;
+        $this->description = $description;
+    }
+
+    public function getNextStatus($actionName)
```

Comment:
Привыкай к типизации.
```suggestion
    public function getNextStatus(string $actionName): string
```